libraries/lib-audio-devices/AudioIOBase.h

```diff
@@ -329,6 +329,7 @@ extern AUDIO_DEVICES_API StringSetting AudioIOHost;
 extern AUDIO_DEVICES_API DoubleSetting AudioIOLatencyCorrection;
 extern AUDIO_DEVICES_API DoubleSetting AudioIOLatencyDuration;
 extern AUDIO_DEVICES_API StringSetting AudioIOPlaybackDevice;
+extern AUDIO_DEVICES_API StringSetting AudioIOPlaybackSource;
```

Comment:
The commit that introduced this Setting object (the third commit) did not remove all uses of the path, although that did happen in a later commit.

I like a sequence of commits that make a sensible stepwise transformation of the source code tree.  I might make a small commit just to introduce this setting and all the rewrites.  Then proceed to the next step.